spdx/document.py

```diff
@@ -189,7 +190,13 @@ class Document(object):
     Represent an SPDX document with these fields:
     - version: Spec version. Mandatory, one - Type: Version.
     - data_license: SPDX-Metadata license. Mandatory, one. Type: License.
+    - name: Name of the document. Mandatory, one. Type: str.
```

Comment:
Do we really need to make this mandatory?

Reply:
I had referred to https://spdx.org/sites/cpstandard/files/pages/files/spdxversion2.1.pdf#page=11 where the `Cardinality` is `Mandatory`. So, what should we do ?

Reply:
leave as is.